add_emoji_gsub.py

```diff
@@ -18,14 +18,22 @@
 
 __author__ = "roozbeh@google.com (Roozbeh Pournader)"
 
-
 import sys
 
-from fontTools import agl
-from fontTools import ttLib
+try:
+    from fontTools import agl
+except ImportError as er:
+    sys.exit("""Failed to import fontTools.
+             Please make sure 'pip install -r requirements.txt' runs succesfull""")
```

Comment:
I think this string will print with a wide indentation at the left. You can use a regular string instead instead of the `"""` string literal. You can add a '\n' if you want to split the two lines

Reply:
better to remove the whole try: except block, IMO

Reply:
+1. Might be worth checking if we're in a venv [if there is an easy way to do that] and then installing requirements.txt early in the build.

Reply:
I removed these lines, and added an exception in the makefile. If a user isn't in a virtual environment the code exits with the following message: `Please start your virtual environment, and run: 'pip install -r requirements.txt'.`